lib/model-access-wrapper.js

```diff
@@ -607,7 +607,22 @@ class ModelAccessWrapper extends CrispHooks {
 	_checkDiffPermissions(permissions, doc, data) {
 		const grant = permissions.getTargetGrant(this.options.permissionsTarget, data);
 		const patch = Update.createFromDiff(doc.data, data);
-		const fields = _.pull(createUpdate(patch).getUpdatedFields(), '_id');
```

Comment:
Previously this was pulling _id.  It no longer is.  Is there a reason for this change?